public/static/locales/es-419/common.json

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
Are these supposed to be empty?

Reply:
This a revert for a breaking change that Peer was testing.